packages/react-composites/src/composites/CallComposite/Strings.tsx

```diff
@@ -173,4 +173,12 @@ export interface CallCompositeStrings {
    * Tooltip text used to inform a user that toggling microphone in lobby is not supported.
    */
   microphoneToggleInLobbyNotAllowed: string;
+  /**
+   * Title used in pane showing all participants in call
+   */
+  peoplePaneTitle: string;
```

Comment:
@prprabhu-ms @PorterNan is this the kind of feature that should go behind beta or not?